<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="9" passed="6" failed="3" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[<img src=/ file://src/main/resources/screenshots/1565100549249.png/>]]>
    </line>
    <line>
      <![CDATA[<img src=/ file://src/main/resources/screenshots/1565100556626.png/>]]>
    </line>
  </reporter-output>
  <suite started-at="2019-08-06T14:09:06Z" name="com.instigate.app.ChatTest" finished-at="2019-08-06T14:09:22Z" duration-ms="16604">
    <groups>
    </groups>
    <test started-at="2019-08-06T14:09:06Z" name="Command line test" finished-at="2019-08-06T14:09:22Z" duration-ms="16604">
      <class name="com.instigate.app.ChatTest">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:00Z" name="setUp" finished-at="2019-08-06T14:09:06Z" duration-ms="5847" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:06Z" name="before" finished-at="2019-08-06T14:09:06Z" duration-ms="390" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="addEmptyPost()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:06Z" name="addEmptyPost" finished-at="2019-08-06T14:09:07Z" duration-ms="967" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at com.instigate.app.ChatTest.addEmptyPost(ChatTest.java:121)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addEmptyPost -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:07Z" name="before" finished-at="2019-08-06T14:09:08Z" duration-ms="770" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="addPost()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:08Z" name="addPost" finished-at="2019-08-06T14:09:09Z" duration-ms="1062" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[<img src=/ file://src/main/resources/screenshots/1565100549249.png/>]]>
            </line>
          </reporter-output>
        </test-method> <!-- addPost -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:09Z" name="before" finished-at="2019-08-06T14:09:09Z" duration-ms="599" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="checkTitle()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:09Z" name="checkTitle" finished-at="2019-08-06T14:09:10Z" duration-ms="291" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTitle -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:10Z" name="before" finished-at="2019-08-06T14:09:10Z" duration-ms="409" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="findFriend()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:10Z" name="findFriend" finished-at="2019-08-06T14:09:10Z" duration-ms="438" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFriend -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:10Z" name="before" finished-at="2019-08-06T14:09:11Z" duration-ms="397" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="sendEmptyMessageAndCheck()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:11Z" name="sendEmptyMessageAndCheck" finished-at="2019-08-06T14:09:12Z" duration-ms="1575" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at com.instigate.app.ChatTest.sendEmptyMessageAndCheck(ChatTest.java:98)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendEmptyMessageAndCheck -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:12Z" name="before" finished-at="2019-08-06T14:09:13Z" duration-ms="410" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="sendMessage()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:13Z" name="sendMessage" finished-at="2019-08-06T14:09:14Z" duration-ms="1029" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sendMessage -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:14Z" name="before" finished-at="2019-08-06T14:09:14Z" duration-ms="417" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="sendMessageAndCheck()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:14Z" name="sendMessageAndCheck" finished-at="2019-08-06T14:09:16Z" duration-ms="1801" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[<img src=/ file://src/main/resources/screenshots/1565100556626.png/>]]>
            </line>
          </reporter-output>
        </test-method> <!-- sendMessageAndCheck -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:16Z" name="before" finished-at="2019-08-06T14:09:18Z" duration-ms="1385" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="signIn()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:18Z" name="signIn" finished-at="2019-08-06T14:09:18Z" duration-ms="453" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method is-config="true" signature="before()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:18Z" name="before" finished-at="2019-08-06T14:09:18Z" duration-ms="384" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- before -->
        <test-method signature="signInEmptyName()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:18Z" name="signInEmptyName" finished-at="2019-08-06T14:09:19Z" duration-ms="417" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertFalse(Assert.java:65)
at org.testng.Assert.assertFalse(Assert.java:75)
at com.instigate.app.ChatTest.signInEmptyName(ChatTest.java:61)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:110)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInEmptyName -->
        <test-method is-config="true" signature="exit()[pri:0, instance:com.instigate.app.ChatTest@2833cc44]" started-at="2019-08-06T14:09:19Z" name="exit" finished-at="2019-08-06T14:09:22Z" duration-ms="3378" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- exit -->
      </class> <!-- com.instigate.app.ChatTest -->
    </test> <!-- Command line test -->
  </suite> <!-- com.instigate.app.ChatTest -->
</testng-results>
